frontend/src/metabase/query_builder/components/QueryModals/QueryModals.tsx

```diff
@@ -168,17 +169,21 @@ export function QueryModals({
       },
     ) => {
       const isDashboardQuestion = _.isNumber(newQuestion.dashboardId());
+      const isModel = newQuestion.type() === "model";
 
       if (isDashboardQuestion) {
         navigateToDashboardQuestionDashboard(
           newQuestion,
           options?.dashboardTabId,
         );
+      } else if (isModel) {
```

Comment:
In `handleSaveModalCreate`, on line 140 we have

```
 if (type === "model" || type === "metric") {
        onCloseModal();
        setQueryBuilderMode("view");
      } 
 ```
 
 We could just use the same logic here, rather than displaying the success toast. I'm not sure if `setQueryBuilderMode("view");` is required, though

Reply:
@sloansparger you worked in this area recently, is `setQueryBuilderMode("view");` required?

Reply:
I decided to match this behaviour

Reply:
Wow. I totally never saw your original message. Sorry about that. I'm not sure I know the answer though.